server/models/Customer.js

```diff
@@ -0,0 +1,49 @@
+const Sequelize = require('sequelize')
+const db = require('../config/database')
+
+const Customer = db.define('customer', {
+  id: {
+    type: Sequelize.INTEGER,
+    autoIncrement: true,
+    primaryKey: true
+  },
+  email: {
+    type: Sequelize.STRING
+  },
+  firstName: {
+    type: Sequelize.STRING
+  },
+  lastName: {
+    type: Sequelize.STRING
+  },
+  phone: {
+    type: Sequelize.STRING
+  },
+  city: {
+    type: Sequelize.STRING
+  },
+  state: {
+    type: Sequelize.STRING
+  },
+  lastOrderPrice: {
+    type: Sequelize.INTEGER
+  },
+  lastOrderDate: {
+    type: Sequelize.STRING
+  }
+});
+
+// Model Hooks
+Customer.beforeBulkCreate((customers) => {
+  for (const customer of customers) {
+    customer.sanitizeEmail()
+  }
+})
+
+// Instance Methods
+Customer.prototype.sanitizeEmail = function () {
+  this.email = this.email.replace(/[&\/\\#,+()$~%'":*?<>{}\s]/g, '').replace('..', '')
```

Comment:
what would happen if the email has `hi@test....com`?
Not too oppressing to solve this edge case, but maybe you can remove the `replace('..', '')` and add `.{2,}` (try to find what this regular expression does)